src/panels/config/integrations/integration-panels/zwave_js/zwave_js-config-router.ts

```diff
@@ -37,17 +52,55 @@ class ZWaveJSConfigRouter extends HassRouterPage {
         tag: "zwave_js-config-dashboard",
         load: () => import("./zwave_js-config-dashboard"),
       },
+      node_config: {
+        tag: "zwave_js-node-config",
+        load: () => import("./zwave_js-node-config"),
+      },
     },
   };
 
+  public connectedCallback() {
+    super.connectedCallback();
+
+    if (!this.hass) {
+      return;
+    }
+    this._loadData();
+  }
+
+  public disconnectedCallback() {
+    super.disconnectedCallback();
+    if (this._unsubs) {
+      while (this._unsubs.length) {
+        this._unsubs.pop()!();
+      }
+      this._unsubs = undefined;
+    }
+  }
+
+  protected firstUpdated(changedProps) {
+    super.firstUpdated(changedProps);
+    this.addEventListener("hass-reload-entries", () => {
+      this._loadData();
+    });
+  }
+
+  protected updated(changedProps: PropertyValues) {
+    super.updated(changedProps);
+    if (!this._unsubs && changedProps.has("hass")) {
+      this._loadData();
+    }
+  }
+
   protected updatePageEl(el): void {
     el.route = this.routeTail;
     el.hass = this.hass;
     el.isWide = this.isWide;
     el.narrow = this.narrow;
     el.configEntryId = this._configEntry;
-    if (this._currentPage === "node") {
-      el.nodeId = this.routeTail.path.substr(1);
+    if (this._currentPage === "node_config") {
+      el.deviceId = this.routeTail.path.substr(1);
+      el.devices = this._deviceRegistryEntries;
```

Comment:
If we only need the device entry registries on this page, we should just subscribe on that page.

Reply:
Did that for a bit of future-proofing, we'll need the device registry information on future pages for managing associations etc as well.

Reply:
It's fine to just do it on the actual page. It's not a lot of overhead.